lib/primer/classify.rb

```diff
@@ -94,19 +94,18 @@ class Classify
 
     class << self
       def call(classes: "", style: nil, **args)
-        extracted_results = extract_hash(args)
-
-        extracted_results[:class] = [
-          validated_class_names(classes),
-          extracted_results.delete(:classes)
-        ].compact.join(" ").strip.presence
-
-        extracted_results[:style] = [
-          extracted_results.delete(:styles),
-          style
-        ].compact.join.presence
-
-        extracted_results
+        extract_css_attrs(args).tap do |extracted_results|
+          classes = +"#{validated_class_names(classes)} #{extracted_results[:class]}"
+          classes.strip!
```

Comment:
```suggestion
          classes = "#{validated_class_names(classes)} #{extracted_results[:class]}".strip
```

I _think_ that running strip here will remove the need of `+"`

Reply:
Yeah, I was trying to avoid the extra allocation.